### DISCUSSION TITLE
Is read error correction performed in RNA mode?

### DISCUSSION THREAD

**Discussion by Alteroldis** [ablab/spades#843] [General]
Hello!
Tell me, it's interesting. It seemed to me that the correction was not working, and the ability to turn it on was removed. If so, please explain why?

**Comment by andrewprzh** [comment]
Hello
Error correction was removed as we noticed it induces additional errors leading to chimeric transcripts when assembling RNA-Seq data. Our experiments show that results without preliminary correction are better.

**Reply by Alteroldis** [reply]
Oh, thanks for the quick response. I thought it was better with correction.
And can you tell me, have you looked at the assembly results with in silico normalization? he fact is that in the rnaspades article you use quite a few reads for assembly and get good results. On the other hand, my output is about several, 3-5, hundreds of thousands of contigs. But I also take all the reads from experiment, and there are about 300-600M of them.

**Reply by Alteroldis** [reply]
Small addition.
A very large number of output contigs have different UTR regions, but the same coding part. About 30% of assembly can be reduce in this way.

**Reply by andrewprzh** [reply]
Sorry for a delayed response.

In fact, we did not try in silico normalization prior to assembly. I heard controversial feedback about this procedure, but never got to test myself.

For such a large dataset it might be reasonable to try and see whether it reduces the amount of similar contigs.  
However, contig cauterization may give more accurate results (as you did, I guess). 

Best
Andrey
